bin/sirius-script-mamba-env-create.bash

```diff
@@ -314,30 +320,30 @@ fi
 
 ##############################################################################
 printf_yellow "Create and prepare mamba enviroment\n"
-printf_yellow_clear "Create new mamba python environment named sirius: "
-if ! mamba env list | grep -q sirius
+printf_yellow_clear "Create new mamba python environment named $ENV_NAME: "
+if ! mamba env list | grep -q $ENV_NAME
 then
-    mamba create --name sirius python=3.9.2 -y
+    mamba create --name $ENV_NAME python=3.9.2 -y
     printf_green "done!\n"
 else
     printf_blue "environment already exists. Skipping...\n"
 fi
 
 ### Activate Environment and do stuff
-printf_yellow_clear "Activate sirius enviroment\n"
-mamba activate sirius
+printf_yellow_clear "Activate $ENV_NAME enviroment\n"
+mamba activate $ENV_NAME
 
-printf_yellow_clear "create link for python-sirius inside the enviroment: "
+printf_yellow_clear "create link for python-${ENV_NAME} inside the enviroment: "
 cd $CONDA_PREFIX/bin
-if ! [ -f python-sirius ]
+if ! [ -f python-$ENV_NAME ]
 then
-    ln -s python3 python-sirius
+    ln -s python3 python-${ENV_NAME}
```

Comment:
this part can be removed, the `python-sirius` link is used in a standardized manner in several scripts/packages